pybamm/parameters/lithium_ion_parameters.py

```diff
@@ -630,6 +636,26 @@ def U(self, sto, T, lithiation=None):
             out.print_name = r"U_\mathrm{p}(c^\mathrm{surf}_\mathrm{s,p}, T)"
         return out
 
+    def H(self, sto):
```

Comment:
this name should be more informative as well since `H` is not a standard abbreviation in battery modeling

Reply:
Also shouldn't this function be calculated from the user-provided lithiation and delithiation OCPs, rather than requiring an additional user input?

Reply:
It has since been renamed to "hysteresis".

I also updated it to calculate from user-provided OCPs. The average OCP is still required as a parameter. Is that okay?